### ISSUE TITLE
Stub generator fails when `using` directive in namespace declaration

### ISSUE THREAD

**Issue by nesterenko-kv** [reactiveui/refit#544]
If `using` directive appear within a namespace declaration stub generator omit them and build wrong file.
Sample:
```cs
namespace Diploma.Client.Core
{
    using System.Threading;
    using System.Threading.Tasks;
    using /*Diploma.Client.Core.*/Requests;
    using /*Diploma.*/Shared.Models;
    using Refit;

    public interface IDiplomaApi
    {
        [Get("/getProfileInfo")]
        Task<User> GetProfileInfoAsync(
            [Header("Authorization")] string accessToken,
            CancellationToken token = default);

        [Post("/signIn")]
        Task<AuthorizationResult> SignInAsync(
            [Body(BodySerializationMethod.Json)] SignInRequest request,
            CancellationToken token = default);

        [Post("/signUp")]
        Task<User> SignUpAsync(
            [Body(BodySerializationMethod.Json)] SignUpRequest request,
            CancellationToken token = default);
    }
}
```

**Comment by Styxxy** [comment]
This is because you used the shortened using declaration (line 6 and 7). If you were to use the full namespace, it wouldn't be a problem.

**Comment by Styxxy** [comment]
@nesterenko-kv I debugged and find that this occurs when you use the shortened using statement (you don't specify the full namespace). I stepped through all the steps, and it is not possible without some bigger work to get this working in Refit.

@onovotny The text of the namespace is taken and put as the root. This doesn't respect that the using statement was defined within a namespace declaration. This is due that the code looks at all Using Directives in the descendants of the Root (and not only in the "root" itself).

It is quite a big job to do this 100% correctly. Either deeper analysis has to be done, rather than only syntax parsing. But I guess this is out of the question. Or another possibility is to make the parsing a lot smarter, in this case the using statements "in the root" have to be parsed and put at the highest level, and using statements within a namespace can be added separately (the mustache template has to be updated as well to get this additional list).

Files impacted (including line numbers):
https://github.com/reactiveui/refit/blob/master/InterfaceStubGenerator.Core/InterfaceStubGenerator.cs#L150
https://github.com/reactiveui/refit/blob/master/InterfaceStubGenerator.Core/GeneratedInterfaceStubTemplate.mustache#L37

Would you like this feature to be implemented, or would you suggest _no_ ?

**Comment by clairernovotny** [comment]
Should be implemented in #1029

**Comment by clairernovotny** [comment]
Please try v6.0-preview.84 and file bugs as you come across them.
